src/visu_config.xsd

```diff
@@ -924,8 +924,13 @@
       <xsd:element name="label" type="label" minOccurs="0" maxOccurs="1"/>
       <xsd:element name="address" type="address" minOccurs="0" maxOccurs="unbounded"/>
     </xsd:sequence>
-    <xsd:attribute ref="value" use="required" />
-    <xsd:attribute name="shortvalue" type="xsd:string" use="optional" />
+    <xsd:attribute ref="value" use="required"/>
+    <xsd:attribute name="shortvalue" type="xsd:string" use="optional">
+      <xsd:annotation>
+      	  <xsd:documentation xml:lang="en">Defines the value which is sent on short push of the button</xsd:documentation>
```

Comment:
I'd say "on **a** short **press** of the button".

The "a" is clearly missing. The difference between push and press is subtle and sources on the net are accepting both with a tendency towards "press" (which is also the word I'd have chosen intuitively).